experiments/exp963_cascade_finemath.py

```diff
@@ -0,0 +1,77 @@
+# nodryrun
+from experiments.datashop.datashop_datasets import datashop_dclm_annotation_subset, datashop_dclm_pretraining_subset
+from experiments.datashop.datashop_runner import DatashopRunner, DatashopRunnerConfig
+from experiments.datashop.default_configs import (
+    default_consolidate_filter_config_kwargs,
+    default_quality_filter_train_config_kwargs,
+)
+from experiments.exp939_finemath import FINEMATH_DATA_FILTER_PROMPT
+from experiments.models import get_model_local_path, llama_3_3_70b_instruct
+from marin.classifiers.utils import CreateDatasetConfig, create_dataset
+from marin.execution.executor import ExecutorStep, output_path_of, this_output_path
+
+FINEMATH_3_POINT_DATA_FILTER_PROMPT = """
```

Comment:
Can we reference the code/paper that has this prompt and comment if we made any modifications?